src/quo/components/text_input.cljs

```diff
@@ -164,8 +165,7 @@
             (merge {:style                   (text-input-style multiline input-style before after)
                     :ref                     ref
                     :placeholder-text-color  (:text-02 @colors/theme)
-                    :color                   (:text-01 @colors/theme)
-                    :underline-color-android :transparent
+                    :underline-color-android "transparent"
```

Comment:
why string?

Reply:
The app was crashing on android, thought that problem may be here - but it was in another file where I was using `:true` instead of `true`. Will discard this line